### PR TITLE
[tool] Adding gr-air-modes

### PR DESCRIPTION
**Pull request by goabonga**

### CODE REVIEW

**Comment by noptrix** [comment]
thank you @goabonga

File: packages/gr-air-modes/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+source=("git+http://github.com/bistromath/gr-air-modes.git"
+        "modes_gui.patch")
+sha512sums=('SKIP'
+            '0b2149ff9b5b6e198380a01018eb385162493ec6ab813d4c83c46f19987a146a0a30fcc9f60c7c8c68b0ac0c18e5b5b08be9bf11000779d008d2483990cbab52')
+
+
+pkgver() {
+  cd "$pkgname"
+
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
```
Comment (FFY00) [inline packages/gr-air-modes/PKGBUILD]: There is a tag available. Could you add it here @noptrix? Thanks.

Comment (goabonga) [reply packages/gr-air-modes/PKGBUILD]: hello @FFY00 i have test to build the package with the tag, but that's not compatible with the current version of gnu-radio

Comment (FFY00) [reply packages/gr-air-modes/PKGBUILD]: What do you mean? I am just saying add the tag too to `pkgver`.

**Pull request merged**